### PR TITLE
Feat/navbar

### PR DESCRIPTION
**Pull request by ronnyas**
This PR replaces the sidebar implemented in the React PWA kit with a always on top, bottom-placed nav bar. 

Placeholders for main sites; overview, send and receive added.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "@radix-ui/react-slot": "^1.0.2",
+    "buffer": "^6.0.3",
```
Comment (asgeir-s) [inline package.json:19]: why?

Comment (ronnyas) [reply package.json:19]: It's used by [src/store/PasskeyStore.ts](https://github.com/asgeir-s/kiip-wallet/blob/b2f8b0c8d4bbec6efe0a932cf0184ed73a5383e1/src/store/PasskeyStore.ts#L42). 


```typescript
function toLocalStorageFormat(
  passkey: PasskeyCredentialWithPubkeyCoordinates,
): PasskeyLocalStorageFormat {
  return {
    rawId: Buffer.from(passkey.rawId).toString('hex'),
    pubkeyCoordinates: passkey.pubkeyCoordinates,
  };
}
```

**Review by asgeir-s — COMMENTED** [review]
Can't we just put the code in the index file (for instance, under sections)? Why have a Page directory, then an index, and a Page file? Also, when there is only an index file in the directory, we should probably just have it as a file on the top level - like `./Page/index.tsx` => `./Page.tsx` , I realize that this is what was left over from the starter we used, but it seams unnecessary 😅
What do you think?

Otherwise, it looks great!

File: package.json (hunk 1 above)
Comment (asgeir-s) [reply package.json:19]: Aha, strange that I did not get an error about it :P

**Review by asgeir-s — APPROVED** [review]

**Pull request merged**